lib/pbench/cli/server/user_delete.py

```diff
@@ -0,0 +1,65 @@
+import click
+from pbench import BadConfig
+from pbench.server import PbenchServerConfig
+from pbench.cli.server import pass_cli_context
+from pbench.cli.server.options import common_options
+from pbench.common.logger import get_pbench_logger
+from pbench.server.database.models.users import User
+from pbench.server.database.database import Database
```

Comment:
Could you put a blank line between lines 1 and 2 and then alphabetize lines 2-8?

Reply:
Ditto for the other two files.